javascript/processingFunctions/replaceTagWithSymbol.js

```diff
@@ -22,8 +22,8 @@ const tagsToReplace = {
   SHARP: "\\#",
   SECT: "§",
 
-  SHORT_SPACE: "@xxx", // will be replaced in processSnippet depending on rendering target (PDF, HTML, etc.)
-  SHORT_SPACE_AND_ALLOW_BREAK: "@yyy", // will be replaced in processSnippet depending on rendering target (PDF, HTML, etc.)
+  SHORT_SPACE: "",
+  SHORT_SPACE_AND_ALLOW_BREAK: "",
```

Comment:
Would it make sense to replace these with "\n" right here?

Reply:
Ah, wait. That's the effect of replacing the tag by "", presumably.

Reply:
Thanks @TobiasWrigstad for the review! Yes, I would have thought "\n" would have been more appropriate here, but it seems to add an extra line break (replaced by two blank lines instead of one). 

Original: replace by "@xxx":
<img width="455" alt="Screenshot 2021-08-08 at 7 08 01 AM" src="https://user-images.githubusercontent.com/60355570/128615969-84765d0d-4952-4dac-9458-2ef18ccf731a.png">

Replace with "\n":
<img width="420" alt="Screenshot 2021-08-08 at 7 08 45 AM" src="https://user-images.githubusercontent.com/60355570/128615975-a54aec8d-197f-4908-9acd-e8c9145f1c8e.png">

Replace  with "":
<img width="468" alt="Screenshot 2021-08-08 at 7 09 13 AM" src="https://user-images.githubusercontent.com/60355570/128615986-360717f6-2012-4df0-98ce-b047affed682.png">